extensions/positron-python/python_files/jedilsp_requirements/requirements.txt

```diff
@@ -1,8 +1,8 @@
 # This file was autogenerated by uv via the following command:
-#    uv pip compile --generate-hashes python_files/jedilsp_requirements/requirements.in
-attrs==24.3.0 \
-    --hash=sha256:8f5c07333d543103541ba7be0e2ce16eeee8130cb0b3f9238ab904ce1e85baff \
-    --hash=sha256:ac96cd038792094f438ad1f6ff80837353805ac950cd2aa0e0625ef19850c308
+#    uv pip compile --generate-hashes --upgrade python_files\jedilsp_requirements\requirements.in
+attrs==24.2.0 \
```

Comment:
I'm a bit surprised this got downgraded 🤷‍♀️

Reply:
I reverted to match upstream and they last updated Dec 2024.